cli.js

```diff
@@ -16,12 +21,20 @@ const MochaFramework = require('./cli-commands/test/test-frameworks/mocha');
     const initCommand = new InitCommand();
     const testCommand = new TestCommand();
     const shapeCommand = new ShapeCommand();
+    const nodeosCommand = new NodeosCommand();
     const deployCommand = new DeployCommand();
     const compileCommand = new CompileCommand();
 
+    // nodeos subcommands
+    const startCommand = new StartCommand();
+    const stopCommand = new StopCommand();
+    const showCommand = new ShowCommand();
+
+    menu.usage('Usage: $0 [command]');
     menu.command(initCommand.template, initCommand.description, InitCommand.defineCommandOptions(initCommand), InitCommand.executeWithContext(initCommand));
     menu.command(testCommand.template, testCommand.description, TestCommand.defineCommandOptions(testCommand), TestCommand.executeWithContext(testCommand, MochaFramework));
     menu.command(shapeCommand.template, shapeCommand.description, ShapeCommand.defineCommandOptions(shapeCommand), ShapeCommand.executeWithContext(shapeCommand));
```

Comment:
We should make everywhere `defineCommandOptions` and `executeWithContext` as one `define` function.

```
const initCommand = new InitCommand();
const testCommand = new TestCommand(MochaFramework);

menu.command(initCommand.template, initCommand.description, initCommand.define());
menu.command(testCommand.template, testCommand.description, testCommand.define());
```

Reply:
updated